includes/elements/container.php

```diff
@@ -125,10 +125,7 @@ protected function get_initial_config() {
 	 */
 	protected function content_template() {
 		?>
-		<# if ( 'boxed' === settings.content_width ) { #>
-			<div class="e-con-inner">
```

Comment:
I have some concerns regarding this change. 
It looks like a fundamental change that can influence BC or 3rd party plugin functionality.